clients/spark-extensions/v3.1/src/test/java/org/projectnessie/spark/extensions/ITNessieStatements.java

```diff
@@ -15,12 +15,17 @@
  */
 package org.projectnessie.spark.extensions;
 
+import org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions;
 import org.junit.jupiter.api.BeforeAll;
 
 public class ITNessieStatements extends AbstractNessieSparkSqlExtensionTest {
 
   @BeforeAll
   protected static void useNessieExtensions() {
```

Comment:
Since you're changing it everywhere now, this can be moved to `AbstractNessieSparkSqlExtensionTest`

Reply:
I thought of moving it, but then `spark-extensions-base` has to depend on `spark-extensions` for using `NessieSparkSessionExtensions` which is a circular dependency. Hence, I didn't move this.

Let me know if there is any other way.

Reply:
Use the class names as `String`s??

Reply:
Added